index.html

```diff
@@ -147,10 +147,15 @@ <h2>Scope</h2>
         </p>
         <ul>
           <li>Addressing errata, ambiguities, and interoperability problems found in previous versions of the Verifiable Credentials Data Model</li>
-          <li>Data models for credentials, presentations, and proofs</li>
-          <li>Registries for the data models</li>
+          <li>A data model for Verifiable Credentials, inclusive of:</li>
+            <ul>
+                <li>credentials and verifiable credentials</li>
+                <li>presentations and verifiable presentations</li>
+                <li>requests for or submissions of verifiable credentials or verifiable presentations</li>
+            </ul>
+          <li>Registries for the data model</li>
```

Comment:
registry for which aspect of the data model? crypto suites?

Reply:
If the DID WG is any indication of the direction this will take, I expect the registry will be for all extension aspects of the VC Data Model. This is the current registry that we have:

https://w3c-ccg.github.io/vc-extension-registry/

We should probably list that as an input document to the VCWG.